core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -206,6 +206,11 @@ public class RestConfig extends AbstractConfig {
       + " runtime request tags in global stats.";
   protected static final boolean METRICS_GLOBAL_STATS_REQUEST_TAGS_ENABLE_DEFAULT = false;
 
+  public static final String SSL_IS_SPIRE_ENABLED_CONFIG = "ssl.spire.enabled";
```

Comment:
Would named listeners be supported here. Basically we need to also be able to open up multiple ssl ports within same application. One for spire and another maybe that is just tls.

Reply:
https://github.com/confluentinc/cc-security-decisions/pull/1402/files

Reply:
Hey @dragonmushu , in the example link you provided, it looks you have 2 port , 1 is TLS, 1 is unsecured. I am not seeing we have 2 ports for 2 TLS.